prow/OWNERS

```diff
@@ -5,8 +5,11 @@ filters:
     approvers:
       - cjwagner
       - fejta
-      # - spxtr (emeritus)
       - stevekuznetsov
+    emeritus_approvers:
```

Comment:
I don't think this is a valid field?
https://github.com/kubernetes/test-infra/blob/d984001ec023c26e1ce65c533fa928936ff2c378/prow/repoowners/repoowners.go#L51-L57

Reply:
It's documented [here](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus) and used [here](https://github.com/kubernetes/kubernetes/blob/6f6890b09e2a8ae7eaaea0a5052d45c3960e1c76/test/OWNERS#L50)... does it really matter, replacing a comment with a non parsed field? At least that one is clearer.

In the future we might want to add it to `repoowners.go` in case someone wants to do something with it.

Reply:
Hmm, I don't see any reference to that field in our repo...
You're right that it shouldn't really matter though.
/lgtm
/approve
/meowvie